### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: src/rootRender.ts [hunk 1]
```diff
@@ -6,8 +6,9 @@ export default function getRootRenderRouter (rootOrigin: any): Router {
   const router = new Router()
   router.all('/', views(path.join(__dirname, 'views'), { extension: 'ejs' }))
   router.use(async (ctx, next) => {
-    console.log('PATHNAME!!!!!!!!!!!!!!!!!!!!!', ctx.path)
-    if (ctx.path === '/') {
+    console.log('ORIGIN!!!!!!!!!!!!!!!!!!')
+    console.log(ctx.origin)
```
Comment (jaxoncreed) [inline src/rootRender.ts]: Remove console.log

File: package.json [hunk 2]
```diff
@@ -15,10 +16,13 @@
...
     "koa-session": "^5.12.2",
+    "koa-static-server": "^1.4.0",
+    "mashlib": "^1.0.0",
     "redis": "^2.8.0",
-    "solid-idp": "^0.1.2",
-    "wac-ldp": "^0.6.0",
-    "websockets-pubsub": "^0.3.2",
+    "solid-idp": "github:inrupt/solid-idp#dev",
+    "solid-server-ts": "github:inrupt/solid-server-ts#dev",
+    "wac-ldp": "github:inrupt/wac-ldp#dev",
```
Comment (jaxoncreed) [inline package.json]: Should we be pushing to master when these are pointing to dev branches?

Comment (michielbdejong) [reply package.json]: fixed, we're only keeping the solid-server-ts as dev from git.

File: src/rootRender.ts (hunk 1 above)
Comment (michielbdejong) [reply src/rootRender.ts]: Done!

**Pull request merged**